src/grid.js

```diff
@@ -282,10 +283,20 @@ Grid.prototype.getEntityRepr = function(
   return content;
 }
 Grid.prototype.getEntityReprs = function(opt_addEndable) {
+  var maximumTetrisSize = 0;
+  for (var i = 0; i < this.storeWidth * this.storeHeight; i++) {
+    var value = this.entities[i];
+    if (value.type == Type.TETRIS) {
+      var height = Math.floor(value.shape.grid.length / value.shape.width);
+      var size = value.shape.width > height ? value.shape.width : height;
+      if (size > maximumTetrisSize)
+          maximumTetrisSize = size;
```

Comment:
Hope you don't mind a very small style nit: indent 2, use curly braces unconditionally.

Change & screenshot looks great otherwise, can insta-merge on style update (or postpone it to unrelated later commit if that's cleaner).